Common/Gfx/Util/D2DBitmapLoader.h

```diff
@@ -17,6 +18,7 @@ class D2DBitmapLoader
 {
 public:
 	static HRESULT LoadBitmapFromFile(const Canvas& canvas, D2DBitmap* bitmap);
+	static HRESULT LoadBitmapFromPluginMeasure(const Canvas& canvas, D2DBitmap* bitmap, PluginImageData& imageData);

```

Comment:
Considering the fact that GUI should ideally not know anything about backend (even if it's already not true for rainmeter), I think it would be better here to declare function like this:
```cpp
HRESULT LoadBitmapFromMemory(const Canvas& canvas, D2DBitmap* bitmap, pixels, width, height);
```